packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -0,0 +1,80 @@
+import { html, ref, slotted } from "@microsoft/fast-element";
+import { Listbox } from "../listbox/listbox";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+import { Combobox } from "./combobox";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(Select:class)} component.
+ * @public
+ */
+export const ComboboxTemplate = html<Combobox>`
+    <template
+        autocomplete="${x => x.autocomplete}"
+        class="${x => (x.open ? "open" : "")} ${x =>
```

Comment:
I think we shouldn't reflect these attrs to classes and should encourage styling based on attr selectors because authoring systems like React can unknowingly remove classes set by the element on the element host.